packages/core/services/google-maps-types.ts

```diff
@@ -554,9 +554,11 @@ export interface MapTypeControlOptions {
 }
 
 export enum MapTypeControlStyle {
-  DEFAULT,
-  DROPDOWN_MENU,
-  HORIZONTAL_BAR,
+  DEFAULT = 0,
+  DROPDOWN_MENU = 2,
+  HORIZONTAL_BAR = 1,
+  INSET = 3,
```

Comment:
INSET and INSET_LARGE are not documented, and doing a quick test, showed that they don't do anything. They should be removed.

Reply:
Done.